assets/js/src/public.jsx

```diff
@@ -6,19 +6,27 @@ import 'parsleyjs';
 const exitIntentEvent = 'mouseleave.mailpoet.form-exit-intent';
 
 jQuery(($) => {
-  window.reCaptchaCallback = function reCaptchaCallback() {
-    $('.mailpoet_recaptcha').each((index, element) => {
-      const recaptcha = $(element);
-      const sitekey = recaptcha.attr('data-sitekey');
-      const container = recaptcha.find('> .mailpoet_recaptcha_container').get(0);
-      const field = recaptcha.find('> .mailpoet_recaptcha_field');
-      let widgetId;
-      if (sitekey) {
-        widgetId = window.grecaptcha.render(container, { sitekey, size: 'compact' });
-        field.val(widgetId);
+  function renderCaptcha(element, iteration) {
+    if (!window.recaptcha || !window.grecaptcha.ready) {
+      if (iteration < 20) {
```

Comment:
Could you please explain to me why is 20 here? Why not some bigger or smaller number? why is 20 the right amount? does it need a comment?